src/authHook.ts

```diff
@@ -49,22 +56,30 @@ const maybeSetAuthToken = async ({
 /**
  * Retrieve an authorization token from AWS CodeArtifact
  *
- * @param {string} domain
- * @param {string} domainOwner
- * @param {string} region
+ * @param {AuthorizationTokenParams} authorizationTokenParams
```

Comment:
I think all parameters in jsdoc should have detailed descriptions and ideally turn on eslint rules to enforce that